src/main/java/logisticspipes/utils/tuples/LPPosition.java

```diff
@@ -99,26 +106,17 @@ public LPPosition moveForward(ForgeDirection dir) {
         return moveForward(dir, 1);
     }
 
-    public LPPosition moveBackward(ForgeDirection dir, double steps) {
-        return moveForward(dir, -1 * steps);
-    }
-
-    public LPPosition moveBackward(ForgeDirection dir) {
-        return moveBackward(dir, 1);
-    }
-
     @Override
     public String toString() {
-        return "(" + getXD() + ", " + getYD() + ", " + getZD() + ")";
+        return "(" + xPos + ", " + yPos + ", " + zPos + ")";
     }
 
     public String toIntBasedString() {
```

Comment:
This says "int-based" but uses the double values 🤷‍♂️  Preserving the existing behavior to prevent regressions.